store/migrations/0001.sql

```diff
@@ -8,7 +8,7 @@ CREATE TABLE IF NOT EXISTS public.payment
     "email"               varchar(255),
     "phone_number"        varchar(255),
     "verification_method" varchar(255),
-    "cvv"                 varchar(255),
+    "cvv"                 varchar(8000),
```

Comment:
because 255 gave `value too long for type character varying(255)` for encrypted cvv